jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java

```diff
@@ -548,7 +553,7 @@
     @UI("#nav-pills-drop")
     public static NavsPillsWithDropdown navsPillsWithDropdown;
     @UI("#navbar-dark-colorscheme")
-    public static NavbarColorScheme navbarDarkColorScheme;
+    public static io.github.com.sections.navbar.NavbarColorScheme navbarDarkColorScheme;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'io.github.com.sections.navbar.NavbarColorScheme' due to existing import 'io.github.com.sections.navbar.NavbarColorScheme'](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4457828)